pkg/kubelet/api/v1alpha1/runtime/api.pb.go

```diff
@@ -458,23 +459,112 @@ func (m *NamespaceOption) GetHostIpc() bool {
 	return false
 }
 
+// SandboxSecurityContext holds security configuration that will be applied to a sandbox.
+type SandboxSecurityContext struct {
+	// The configurations for the sandbox's namespaces.
+	// This will be used only if the PodSandbox uses namespace for isolation.
+	NamespaceOptions *NamespaceOption `protobuf:"bytes,1,opt,name=namespace_options,json=namespaceOptions" json:"namespace_options,omitempty"`
+	// Optional SELinux context to be applied. Only for sandbox. Containers should
+	// confirm to SecurityContext.SelinuxOptions.
+	SelinuxOptions *SELinuxOption `protobuf:"bytes,2,opt,name=selinux_options,json=selinuxOptions" json:"selinux_options,omitempty"`
```

Comment:
It's hard to ensure the field name for both grpc and golang conventions. Some options here

- Option 1: Named `SELinuxOptions` with json `sELinuxOptions`
- Option 2: Named `SeLinuxOptions` with json `seLinuxOptions`
- Option 3: Named `SelinuxOptions ` with json `selinuxOptions`

Option 3 seems most comfortable to me. @pmorie What's your opinion?